ui/app/src/components/CompareVersionsDialog/translations.ts

```diff
@@ -20,6 +20,12 @@ export const translations = defineMessages({
   backToSelectRevision: {
     id: 'compareVersionsDialog.back.selectRevision',
     defaultMessage: 'Back to select revision'
+  },
+  compareXml: {
+    defaultMessage: 'XML'
+  },
+  compareContent: {
+    defaultMessage: 'Content'
```

Comment:
If possible, use <FormattedMessage />

Reply:
@rart EnhancedDialog's header title prop is of type string, should I update the dialog for it to support ReactNode?